lib/Image/Adapter/Imagick.php

```diff
@@ -758,6 +758,11 @@ public function addOverlay(mixed $image, int $x = 0, int $y = 0, int $alpha = 10
                     $frame->compositeImage($newImage, $compositeValue, $x, $y);
                 }
             } else {
+                $imageColorspace = $newImage->getImageColorspace();
```

Comment:
I think we can safely remove this extra variable here?
WDYT? 
We do nothing else with it than in this one check. 

Rest looks fine to me.

Reply:
Updated